marklogic-client-api/src/main/java/com/marklogic/client/DatabaseClient.java

```diff
@@ -225,6 +226,13 @@
    * @return the connection type
    */
   ConnectionType getConnectionType();
+  
+  /**
+   * Checks if the connection is valid.
+   * @return {@link ConnectionResult}  with a connected property of true or false. 
```

Comment:
Could the @link be removed if the ConnectionResult were a nested interface?  (See comment below)

Reply:
Changed the link to point the interface ConnectionResult.